src/js/pages/HowItWorks.jsx

```diff
@@ -289,6 +292,30 @@ class HowItWorks extends Component {
     });
   };
 
+  sendHowItWorksSlideEvent (stepIndex) {
+    const { location: { pathname: currentPathname } } = window;
+    const { pageName, pageType } = lookupPageNameAndPageTypeDict(currentPathname);
+    TagManager.dataLayer({
+      dataLayer: {
+        event: 'action',
+        pageDetails: {
+          pageName,
+          pageType,
+          pathname: currentPathname,
+        },
+        actionDetails: {
+          eventType: 'slideChange',
```

Comment:
I really like the value 'slideChange' -- I added that to the [Property Name spreadsheet](https://docs.google.com/spreadsheets/d/1KO4_AOmddKvIpA79XrZdaVBdRkcP1HitqrvQkCo2A34/edit?pli=1&gid=1828322090#gid=1828322090).
The variable name should be 'actionType' instead of 'eventType'.